docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -79,36 +69,43 @@ The default value is `false`, which is the same as omitting the property entirel
+
 > [!NOTE]
->
-> - For clarity, in this section we call the message target the host *page*, but strictly speaking the messages are going to the [Runtime](../testing/runtimes.md) in the task pane (or the runtime that is hosting a [function file](/javascript/api/manifest/functionfile)). The distinction is only significant in the case of cross-domain messaging. For more information, see [Cross-domain messaging to the host runtime](#cross-domain-messaging-to-the-host-runtime).
-> - The dialog box can't communicate with the host page in the task pane unless the Office JavaScript API library is loaded in the page. (Like any page that uses the Office JavaScript API library, script for the page must initialize the add-in. For details, see [Initialize your Office Add-in](initialize-add-in.md).)
+> For clarity, in this section we call the message target the host *page*, but strictly speaking the messages are going to the [Runtime](../testing/runtimes.md) in the task pane (or the runtime that is hosting a [function file](/javascript/api/manifest/functionfile)). The distinction is only significant in the case of cross-domain messaging. For more information, see [Cross-domain messaging to the host runtime](#cross-domain-messaging-to-the-host-runtime).
 
-Code in the dialog box uses the [messageParent](/javascript/api/office/office.ui#office-office-ui-messageparent-member(1)) function to send a string message to the host page. The string can be a word, sentence, XML blob, stringified JSON, or anything else that can be serialized to a string or cast to a string. The following is an example.
+The following example shows how to initialize Office JS and send a message to the host page.
 
 ```js
-if (loginSuccess) {
+Office.onReady(function() {
+   // Add any initialization code for your dialog here.
+});
+
+// Called when dialog signs in the user.
+function userSignedIn() {
     Office.context.ui.messageParent(true.toString());
 }
 ```
 
-> [!IMPORTANT]
->
-> - The `messageParent` function is one of *only* two Office JS APIs that can be called in the dialog box.
-> - The other JS API that can be called in the dialog box is `Office.context.requirements.isSetSupported`. For information about it, see [Specify Office applications and API requirements](specify-office-hosts-and-api-requirements.md). However, in the dialog box, this API isn't supported in volume-licensed perpetual Outlook 2016 (that is, the MSI version).
-
-In the next example, `googleProfile` is a stringified version of the user's Google profile.
+The next example shows how to return a JSON string containing profile information.
 
 ```js
-if (loginSuccess) {
-    Office.context.ui.messageParent(googleProfile);
+function userProfileSignedIn(profile) {
+    const profileMessage = {
+        "name": profile.name,
+        "email": profile.email,
+    };
+    Office.context.ui.messageParent(JSON.stringify(profileMessage));
 }
 ```
 
+> [!NOTE]
```

Comment:
I think this note can be converted to be a normal paragraph in this section. I noted several "alerts" (ie. Note, Tip, Important) in this topic. Consider reviewing and seeing if any should be converted to normal. It's very "choppy" as is.

Reply:
fixed.